packages/suite/src/components/wallet/TransactionItem/index.tsx

```diff
@@ -169,23 +168,40 @@ const TransactionItem = React.memo(
         const [limit, setLimit] = useState(0);
         const isTokenTransaction = tokens.length > 0;
         const isUnknown = isTxUnknown(transaction);
-        const isExpandable = isTokenTransaction
-            ? tokens.length - DEFAULT_LIMIT > 0
-            : targets.length - DEFAULT_LIMIT > 0;
-        const toExpand = isTokenTransaction
-            ? tokens.length - DEFAULT_LIMIT - limit
-            : targets.length - DEFAULT_LIMIT - limit;
         const useFiatValues = !isTestnet(transaction.symbol);
-        const hasSingleTargetOrTransfer = !isUnknown && targets.length + tokens.length === 1;
-        const showFeeRow = !isUnknown && type !== 'recv' && transaction.fee !== '0';
+        const useSingleRowLayout =
+            !isUnknown && (targets.length + tokens.length === 1 || transaction.type === 'self');
+
+        const showFeeRow =
+            !isUnknown &&
+            isTokenTransaction &&
```

Comment:
Too strong condition probably? It changes the behaviour for BTC too. It's not intentional, right?

production:
<img width="956" alt="image" src="https://user-images.githubusercontent.com/3729633/154978155-17d8ea0d-dad0-4ed7-97f3-52e6f155d9b3.png">

this branch:
<img width="961" alt="image" src="https://user-images.githubusercontent.com/3729633/154978310-51fca934-c655-4f7f-9a19-7b70f6382f23.png">

Reply:
Huh interesting, I assume I wanted to fix a behaviour where in sent to self tx there is the same amount rendered twice - 1. as output amount 2. as fee, since sent to self transaction in Suite has amount always equal to fee, but I probably didn't realize it was not a bug I introduced with cardano and in the process of trying to fix it I introduced this change 😓  I'll just revert it. 

<img width="841" alt="Screenshot 2022-02-21 at 17 40 42" src="https://user-images.githubusercontent.com/6961901/154996168-93397ca0-f449-47b9-a12d-ce59776218a5.png">

Reply:
should be fixed in https://github.com/trezor/trezor-suite/pull/4144/commits/e661404f09682ec6caa6bd5a17a4ca7abaa378cf